app/javascript/leftSidebar/TagsFollowed.jsx

```diff
@@ -0,0 +1,27 @@
+import { h } from 'preact';
+import PropTypes from 'prop-types';
+import { tagPropTypes } from '../src/components/common-prop-types';
+
+export const TagsFollowed = ({ tags = [] }) => {
```

Comment:
We are using Preact 8.x which currently does not support [Fragments](https://reactjs.org/docs/fragments.html#short-syntax) (they do in Preact X). Because of this, a dummy container in markup needs to be created so that the Preact component `<TagsFollowed />` can be mounted.